PEPRest/pom.xml

```diff
@@ -14,15 +14,14 @@
 	<version>2.0-SNAPSHOT</version>
 
 	<properties>
-		<jclouds.version>1.9.1</jclouds.version>
-		<springfox.version>2.2.2</springfox.version>
-		<swagger.version>1.5.3</swagger.version>
```

Comment:
Is swagger still  there? I use its UI to start a request

Reply:
yes it's still there.